### ISSUE TITLE
Accessibility: [Keyboard] Focus loss after pressing "Tab" from Health Bot bar.

### ISSUE THREAD

**Issue by ericmaino** [microsoft/BotFramework-WebChat#3085]
![image](https://user-images.githubusercontent.com/1005609/78628461-0cbed480-7849-11ea-84ba-552e52c12c1f.png)

**Expected**
After pressing Tab from Health Bot address bar  (kphelpbot.azurefd.net), the focus should come to first actionable element in DOM which is call 911

**Actual**
The focus is not visible and is present on the non-actionable container (highlighted by red rectangle in the screenshot below). 

**WCAG Violation**
SC 2.4.7:  Focus Visible
https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html

**Comment by corinagum** [comment]
These are accessibility issues of v3 of Web Chat, which has been deprecated and we no longer support unless there are security risks. For accessible Web Chat, please [migrate to v4](https://github.com/Microsoft/BotFramework-WebChat#migrating-from-web-chat-v3-to-v4).

**Comment by compulim** [comment]
This should be fixed in Web Chat v4.
